docker/Dockerfile.localnet

```diff
@@ -8,9 +8,13 @@ RUN mv /app/test-config.yaml /app/config.yml
 RUN chown -R tendermint /app && chown -R tendermint /go
 
 USER tendermint
+RUN /bin/bash -c 'ARCH=`uname -m` && \
+    if [ "$ARCH" == "aarch64" ]; then export ARCH="arm64"; else export ARCH="amd64"; fi && \
+    export URL=https://github.com/TomWright/dasel/releases/latest/download/dasel_linux_$ARCH && \
+    wget $URL -O /go/bin/dasel && chmod +x /go/bin/dasel'
 
-RUN go install github.com/tomwright/dasel/cmd/dasel@master
-RUN ignite chain init
+#RUN go install github.com/tomwright/dasel/cmd/dasel@master
+RUN /bin/bash -c 'ignite chain init --skip-proto -y'
```

Comment:
I was facing longer wait times for building containers. So I changed a bit things
1. download dasel binary instead on building it
2. Skip proto generation
3. the latest ignite(v0.24.0) image requires to use v1 config which is different than we use right now. `-y` will convert current `test-config.yml` to v1 format. Since this is happening inside container this wouldn't affect code base on our disk.